main.js

```diff
@@ -285,3 +285,21 @@ listProject.forEach((project) => {
   work.append(portfolioSnapshot);
   index += 1;
 });
+
+/* Form Validation */
+const form = document.getElementById('form');
+const error = document.getElementById('error');
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const { email } = form.elements;
+  if (email.validity.typeMismatch
+    || email.value.toLowerCase() !== email.value
+    || !email.validity.valid) {
+    error.textContent = 'Invalid email. Please ensure you have the correct email address; also, check that the content of the email field is in lowercase. Thank you.';
```

Comment:
- [ ] OPTIONAL
I think you should display specific error message for a specific validation error message. When I enter an email in capitals, I get the message that my email is incorrect, though it appears correct except that it is in block letters. Kindly endeavor to improve upon the same.

![image](https://user-images.githubusercontent.com/12745474/202291688-517807ed-507c-4431-a87a-2bbd7198d0c5.png)